src/plugins/vis_types/xy/public/editor/components/options/metrics_axes/label_options.tsx

```diff
@@ -34,6 +40,16 @@ function LabelOptions({ axisLabels, axisFilterCheckboxName, setAxisLabel }: Labe
   );
 
   const rotateOptions = useMemo(getRotateOptions, []);
+  const multilayerAxisTooltipText = disableSingleLayerAxisControls
+    ? i18n.translate(
+        'visTypeXy.controls.pointSeries.categoryAxis.axisLabelsOptionsMultilayer.disabled',
+        {
+          defaultMessage:
+            'This option is only available for categorical axes or the legacy time axis (see Advanced Settings -> visualization:useLegacyTimeAxis)',
```

Comment:
Same as Lens, the proposal is a simpler text that doesn't mention the legacy setting

Reply:
updated the copy here: https://github.com/elastic/kibana/pull/117828/commits/9b9540ac00ba4163436a534dc008bd5e60d6bc6b